tests/test_mace_fitting.py

```diff
@@ -0,0 +1,54 @@
+import yaml, os, shutil
+import time
+from pathlib import Path
+import mace, json
+from wfl.fit.mace import fit
+from wfl.configset import ConfigSet
+import ase.io
+import pytest
+
+os.environ["WFL_MACE_FIT_COMMAND"] = "python /raven/u/hjung/Softwares/mace/scripts/run_train.py"
```

Comment:
Please remove things that are hard-wired to your specific local setup. If you need that env var set when you run your pytests, just set it in your shell.  When it runs pytest the github CI workflow will either set that env var or use a version of mace that has the executable in path.